primitives/account/src/logs.rs

```diff
@@ -0,0 +1,364 @@
+use crate::Receipt;
+use beserial::Serialize as BeSerialize;
+use nimiq_hash::Blake2bHash;
+use nimiq_keys::Address;
+use nimiq_primitives::coin::Coin;
+use nimiq_transaction::account::htlc_contract::{AnyHash, HashAlgorithm};
+
+#[derive(Debug, PartialEq, Eq, Clone)]
+#[cfg_attr(feature = "serde-derive", derive(serde::Serialize, serde::Deserialize))]
+// Renaming affects only the struct names and thus their tag, the "type" field.
+#[serde(rename_all = "kebab-case", tag = "type")]
```

Comment:
Shouldn't this be `camelCase`? Maybe we can change it here and avoid adding it for all of the members?